src/Storage/null.cpp

```diff
@@ -0,0 +1,14 @@
+//  Copyright © 2024
+//  Author YX.E.Z yixuanzhou@sjtu.edu.cn
+//  All rights reserved.
+//
+//  This code is licensed under the Creative Commons Attribution-NonCommercial 4.0 International License.
```

Comment:
Could this be a more permissive licence, e.g. Apache or MIT? Creative Commons License is not recommended for software.